controllers/storagecluster/exporter.go

```diff
@@ -1107,3 +1117,39 @@ func (r *StorageClusterReconciler) deleteMetricsExporterCephClient(namespace str
 
 	return nil
 }
+
+func getRookCephHostNetwork(instance *ocsv1.StorageCluster) bool {
```

Comment:
This function returns a bool, it should not be called `get` as it does not get anything. Please consider renaming it to something like `shouldUseHostNetworking`